openapi-contracts/src/test/java/com/sinch/sdk/domains/verification/models/dto/v1/webhooks/VerificationResponseEventDtoTest.java

```diff
@@ -3,49 +3,37 @@
 import com.adelean.inject.resources.junit.jupiter.GivenTextResource;
 import com.adelean.inject.resources.junit.jupiter.TestWithResources;
 import com.fasterxml.jackson.core.JsonProcessingException;
-import com.sinch.sdk.BaseTest;
+import com.sinch.sdk.domains.verification.adapters.VerificationBaseTest;
 import com.sinch.sdk.domains.verification.models.v1.start.request.PhoneCallSpeech;
-import com.sinch.sdk.domains.verification.models.v1.webhooks.FlashCallRequestEventResponse;
-import com.sinch.sdk.domains.verification.models.v1.webhooks.FlashCallRequestEventResponseFlashCallContent;
-import com.sinch.sdk.domains.verification.models.v1.webhooks.PhoneCallRequestEventResponse;
-import com.sinch.sdk.domains.verification.models.v1.webhooks.PhoneCallRequestEventResponsePhoneCallContent;
-import com.sinch.sdk.domains.verification.models.v1.webhooks.SmsRequestEventResponse;
-import com.sinch.sdk.domains.verification.models.v1.webhooks.SmsRequestEventResponseSmsContent;
 import com.sinch.sdk.domains.verification.models.v1.webhooks.VerificationEventResponseAction;
+import com.sinch.sdk.domains.verification.models.v1.webhooks.VerificationRequestEventResponseFlashCall;
+import com.sinch.sdk.domains.verification.models.v1.webhooks.VerificationRequestEventResponsePhoneCall;
+import com.sinch.sdk.domains.verification.models.v1.webhooks.VerificationRequestEventResponseSms;
 import java.util.Collections;
 import org.json.JSONException;
 import org.junit.jupiter.api.Test;
 import org.skyscreamer.jsonassert.JSONAssert;
 
 @TestWithResources
-public class VerificationResponseEventDtoTest extends BaseTest {
+public class VerificationResponseEventDtoTest extends VerificationBaseTest {
 
-  public static PhoneCallRequestEventResponse expectedPhoneCallRequestEventResponseDto =
-      PhoneCallRequestEventResponse.builder()
+  public static VerificationRequestEventResponsePhoneCall expectedPhoneCallRequestEventResponseDto =
+      VerificationRequestEventResponsePhoneCall.builder()
           .setAction(VerificationEventResponseAction.ALLOW)
-          .setCallout(
-              PhoneCallRequestEventResponsePhoneCallContent.builder()
-                  .setCode("4567")
-                  .setSpeech(PhoneCallSpeech.builder().setLocale("the locale").build())
-                  .build())
+          .setCode("4567")
+          .setSpeech(PhoneCallSpeech.builder().setLocale("the locale").build())
```

Comment:
I asked myself the question but seems it could be an issue in case of a new field will be added Speech schema.
Imagine in this context a new field named "code" is added to the speech schema: it will cause conflicts with the "code" defined at root VerificationRequestEventResponsePhoneCall level.

Currently delegation is used in allOf context only  to create base class + inherited ones to "hide" and reduce subfield complexity. 
In this context we know we won't have conflicts by using delegation pattern